packages/ns-api/files/ns.dpi

```diff
@@ -37,7 +37,11 @@ if cmd == 'list':
             'applications': 'str',
             'protocols': 'str'
         },
-        'list-devices': {}
+        'list-devices': {},
+        'list-popular': {
+            'limit': 32,
+            'page': 32
```

Comment:
Is this really needed?

Reply:
Not really, I thought of removing it too, however since we're gonna change the UI sometimes in the future, we can still use the function without additional changes

Reply:
ok